channels.json

```diff
@@ -12,7 +12,7 @@
     },
     {
       "name": "Quần Cam",
-      "avatar_url": "",
+      "avatar_url": "https://quancam.net/assets/images/logo-front.png",
```

Comment:
Sorry but we do not support loading avatars from an URL yet, could you pls add this file to 'public/avatars' folder